Directory.Build.props

```diff
@@ -9,7 +9,7 @@
         <PackageReference Include="StyleCop.Analyzers" Version="1.2.0-beta.321" PrivateAssets="all" />
         <AdditionalFiles Include="$(SolutionDir)stylecop.json" Link="stylecop.json" />
 
-        <PackageReference Include="SonarAnalyzer.CSharp" Version="8.14.0.22654" />
+        <PackageReference Include="SonarAnalyzer.CSharp" Version="8.17.0.26580" />
```

Comment:
This is an attempt to fix a false positive in Sonar Analyzer: https://github.com/perlang-org/perlang/pull/209/checks?check_run_id=3709390145

(Upstream issue is https://github.com/SonarSource/sonar-dotnet/issues/3105)